simple/entities/main.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+from absl import flags
+from absl import app
+import constants
+from importer import SimpleEntitiesImporter
+
+# For importing util
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(1, os.path.join(_CODEDIR, "../"))
+
+from util import dc_client as dc
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_enum("entity_type", None, ["Country", "City"],
```

Comment:
Should we make this the first column name?  Might be nice to have the CSV completely self-contained, no?

CC @shifucun

Reply:
Not sure if we want user to enter type schema, either as command line or as a column. Even I don't know what the type for zip code is exactly :). 

I feel the user flow is still an open ended question. And later on, maybe just have a script to focus on resolving! Otherwise, this folder will becoming more and more complicated and can only be used by us.

Reply:
SG